src/components/profile/UserProfile.js

```diff
@@ -372,31 +390,41 @@ class UserProfile extends Component {
                     <MainContainer marginTop={15}>
                         <SectionHeader>
                             <SectionTitle>Classes Taken</SectionTitle>
-                            <SearchIcon source={MagnifyingIcon} />
+                            {/* <SearchIcon source={MagnifyingIcon} /> */}
+                            <ModifyIcons isDisplayed={!this.props.isReadOnly}>
+                                <AddClassesTakenModal
+                                    userID={this.state.userID}
+                                    token={this.state.token}
+                                    getCurrentUser={this.getCurrentUser} />
+                            </ModifyIcons>
                         </SectionHeader>
                         <LabelContainer>
                             {classesTaken}
                         </LabelContainer>
-                        <ModalsContainer isDisplayed={!this.props.isReadOnly}>
+                        {/* <ModalsContainer isDisplayed={!this.props.isReadOnly}>
                             <AddClassesTakenModal userID={this.state.userID} token={this.state.token} getCurrentUser={this.getCurrentUser} />
                             <RemoveClassesTakenModal userID={this.state.userID} token={this.state.token} getCurrentUser={this.getCurrentUser} />
-                        </ModalsContainer>
+                        </ModalsContainer> */}
 
                     </MainContainer>
 
                     <MainContainer marginTop={15}>
                         <SectionHeader>
                             <SectionTitle>Classes enrolled in </SectionTitle>
+                            <ModifyIcons isDisplayed={!this.props.isReadOnly}>
+                                <RemoveCurrentClassModal userID={this.state.userID} token={this.state.token} getCurrentUser={this.getCurrentUser} />
```

Comment:
can we add more spacing between the add and remove class icons or make them larger? I mistakenly press the wrong button bc both fit under my finger